Apps/APN/App.js

```diff
@@ -0,0 +1,208 @@
+import React, {useEffect, useRef, useState} from 'react';
+import { ActivityIndicator,Linking,StyleSheet} from 'react-native';
+import Login from './components/Login';
+import { NavigationContainer, useNavigationContainerRef } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { CustomerIO, CustomerioConfig, CioLogLevel, CustomerIOEnv } from "customerio-reactnative";
+import Dashboard from './components/Dashboard';
+import CustomDataScreen from './components/CustomDataScreen';
+import SettingsScreen from './components/SettingsScreen'
+import Env from "./env";
+import CioManager from './manager/CioManager';
+import CioKeyValueStorage from './manager/KeyValueStorage';
+import Deeplinks from './components/Deeplink';
+
+const Stack = createNativeStackNavigator();
+
+export default function App() {
+
+  const [firstScreen, setFirstScreen] = useState(undefined)
+  const [loading, setLoading] = useState(true);
+const [isScreenTrackEnabled, setIsScreenTrackEnabled] = useState(null)
+const [isDeviceAttrTrackEnabled, setIsDeviceAttrTrackEnabled] = useState(null)
+const [isDebugModeEnabled, setIsDebugModeEnabled] = useState(null)
+const [bgDelayValue, setBgDelayValue] = useState(null)
+const [bgTasksValue, setBgTasksValue] = useState(null)
+const [trackingUrl, setTrackingUrl] = useState(null)
```

Comment:
Does the app need these properties? 

I think that these variables are used for CIO SDK initialization. If that is true, can we instead encapsulate SDK initialization inside of a function in this file? 

Something like this is what I was expecting to see: 

```js
initialiseCioPackage() {
    const keyStorageObj = new CioKeyValueStorage()
    const bgDelayValue  = await keyStorageObj.getBGQSecondsDelay()
    const bgTasksValue = await keyStorageObj.getBGQMinTasksInQueue()
    const screenTrackValue = await keyStorageObj.getScreenTrack()
    const deviceAttrValue = await keyStorageObj.getDeviceAttributesTrack()
    const debugModeValue = await keyStorageObj.getDebugModeConfig()
    const trackUrl = await keyStorageObj.getTrackingUrl()

    const configuration = new CustomerioConfig()
    configuration.logLevel = isDebugModeEnabled === null ? CioLogLevel.debug : isDebugModeEnabled
    configuration.autoTrackDeviceAttributes = isDeviceAttrTrackEnabled === null ? true : isDeviceAttrTrackEnabled 
    configuration.backgroundQueueMinNumberOfTasks = bgTasksValue
    configuration.backgroundQueueSecondsDelay = bgDelayValue
    if(trackingUrl != null) {
      configuration.trackingApiUrl = trackingUrl
    }

    const env = new CustomerIOEnv()
    env.siteId = Env.siteId
    env.apiKey = Env.apiKey

    const cioManager = new CioManager()
    cioManager.initializeCio(env, configuration)
}
```